src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentivesMigrationService.kt

```diff
@@ -99,8 +99,8 @@ class IncentivesMigrationService(
     )
   }
 
-  fun migrateIncentivesForPage(context: MigrationContext<IncentivesPage>) =
-    nomisApiService.getIncentivesBlocking(
+  suspend fun migrateIncentivesForPage(context: MigrationContext<IncentivesPage>) =
+    nomisApiService.getIncentives(
```

Comment:
I've therefore all the `Blocking` versions of service methods since the listeners are also now async